src/systems/filecoin_mining/storage_mining/storage_miner_actor_code.go

```diff
@@ -626,7 +641,12 @@ func (a *StorageMinerActorCode_I) _rtVerifySurprisePoStOrAbort(rt Runtime, onCha
 		addr.Serialize_Address_Compact(rt.CurrReceiver()),
 	)
 
-	if !postRand.Verify(postRandomnessInput, info.WorkerVRFKey()) {
+	minerKey, err := info.GetWorker(rt.CurrEpoch()).GetKey()
+	if err != nil {
+		rt.AbortStateMsg("Couldn't get worker key to verify surprise PoSt.")
+	}
```

Comment:
Can this ever fail? If not, let's make it an assert. We'll need to decide later whether assertion failures should crash the VM or not, but in any case, we're currently distinguishing between errors that might legitimately arise from user input (as `rt.Abort`) vs. errors that indicate a software implementation bug (as `util.Assert`).